docs/src/manual/operations.md

```diff
@@ -113,6 +113,7 @@ solver (including SCS and Mosek).
 | `opnorm(x, 2)` (`operatornorm(x)`)| max of singular values of $x$      | convex       | not monotonic |                        |
 | `eigmax(x)`                       | max eigenvalue of $x$              | convex       | not monotonic |                        |
 | `eigmin(x)`                       | min eigenvalue of $x$              | concave      | not monotonic |                        |
+| `rootdet(X)`                      | root-determinant of the matrix X   | concave      | not monotonic |                        |
```

Comment:
Maybe mention somewhere that by root we mean the `n`-th root ?

Reply:
CVX calls this `det_rootn`. Is that better?

Reply:
`rootdet` is more consistent with `LinearAlgebra.logdet`. We could also just support `det(Q)^(1/n)`, that will be needed anyway for the MOI wrapper to support `JuMP.@objective(model, Max, det(Q)^(1/n))`

Reply:
We decided to go for `rootdet` in the JuMP-dev call, we can add `rootdet` as operator to JuMP later if needed